scripts/ayp.coffee

```diff
@@ -335,8 +339,11 @@ class AYPStrip
 
           @compositeImage frame, char, left, top
 
+          isKenny = char.matchchar.match('default') == null
```

Comment:
Same as above, can this be exact?

And since we're doing it twice, why not declare it as a function inside this method near the `scrambleLines` init and just call it.